opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -240,20 +244,6 @@ message Profile {
   // This field is required.
   bytes profile_id = 17;
 
-  // attributes is a collection of key/value pairs. Note, global attributes
-  // like server name can be set using the resource API. Examples of attributes:
-  //
-  //     "/http/user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36"
-  //     "/http/server_latency": 300
-  //     "abc.com/myattribute": true
-  //     "abc.com/score": 10.239
-  //
-  // The OpenTelemetry API specification further restricts the allowed value types:
-  // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/README.md#attribute
-  // Attribute keys MUST be unique (it is not allowed to have more than one
-  // attribute with the same key).
-  repeated opentelemetry.proto.common.v1.KeyValue attributes = 18;
```

Comment:
I think `Profile.attributes` and `Profile.attribute_table` serve different purpose.

`Profile.attributes` is the list of attributes for the profile itself.

`Profile.attribute_table` is a table referenced by Sample.attribute_indices.

I'm not sure we can remove either of them. At least I don't think this is a typo or an accidental duplication.

Reply:
https://github.com/open-telemetry/opentelemetry-proto/pull/606/commits/bcfb44e09110272a3016c951b8b5c88ff15b3191 adds a list of indices into `attributes_table`  - does this address your concern?

Reply:
Yes, this looks good. AFAICT this brings all uses of attributes into consistency as they are not all referenced via the indices. One note is that in this case this indirection optimization has little value since, unlike other messages, there is no repetition / nesting for Profile, plus the indirection potentially makes debugging a little harder (such as dumping the profile as a proto and reading the profile attributes in the dump), but it's hard to weigh whether hypothetically easier debugging is worth inconsistency, so the change LGTM.